web/webpack.plugins.js

```diff
@@ -11,6 +11,11 @@ module.exports = [
     filename: 'index.html',
     inject: true,
   }),
+  new HtmlWebpackPlugin({
```

Comment:
Is the change to `webpack.plugins.js` needed? I tried it without this change it and it still seems to work.

Reply:
@chamkank Yes, we need it to make `404.html` in `docs/`.

If you delete `404.html`, run `yarn build`, and run `yarn serve`, it shouldn't work because `404.html` wouldn't be generated. Also if you change `index.html` then this would be needed to update `404.html` as well.

Reply:
> Also if you change index.html then this would be needed to update 404.html as well.

Ah I didn't think of that